ooniapi/services/reverseproxy/scripts/40-start-prom-exporter.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# This script is used to start the prometheus exporter for nginx.
+# More about the exporter here: 
+# https://github.com/nginx/nginx-prometheus-exporter
+
+nginx-prometheus-exporter --nginx.scrape-uri=http://localhost:8080/stub_status &
```

Comment:
Is there the chance this process might crash and then it's not brought up again because it's run only using RUN in the dockerfile?

Reply:
Suggestion applied, it worked well with `supervisord`

However we had to replace the original `docker-entrypoint.sh` with our own because changing the `CMD` of the docker file was preventing the original script from running the template set up required to create our configuration file, see: 

https://github.com/nginxinc/docker-nginx/blob/cffeb933620093bc0c08c0b28c3d5cbaec79d729/mainline/debian/docker-entrypoint.sh